mxd/3.1_MXD_tutorial1_dataexchange.md

```diff
@@ -0,0 +1,181 @@
+# Provide and consume data
+
+As described in the introduction, a data exchange between Bob (Data Provider) and Alice (Data Consumer) is to be tested.
```

Comment:
```suggestion
As described in the introduction, Bob (Provider) wants to exchange data with Alice (Consumer).
```

Reply:
Why dont we call the participants Data Provider and Data Consumer? These are the terms stated in the new regulatory framework.